src/libsystemd/sd-json/sd-json.c

```diff
@@ -4089,6 +4090,42 @@ _public_ int sd_json_buildv(sd_json_variant **ret, va_list ap) {
                         break;
                 }
 
+                case _JSON_BUILD_STRING_ORDERED_SET: {
+                        OrderedSet *set;
+
+                        if (!IN_SET(current->expect, EXPECT_TOPLEVEL, EXPECT_OBJECT_VALUE, EXPECT_ARRAY_ELEMENT)) {
+                                r = -EINVAL;
+                                goto finish;
+                        }
+
+                        set = va_arg(ap, OrderedSet*);
+
+                        if (current->n_suppress == 0) {
+                                _cleanup_free_ char **sorted = NULL;
+
+                                sorted = ordered_set_get_strv(set);
```

Comment:
Not a big matter, but `ordered_set_get_strv()` does not sort the result. So, please rename the variable.